File: scala_day01/scala/com/oxysa/forexpression/Demo02.scala
package com.oxysa.forexpression

/**
  * @author Yummy
  * @mail 873387751@qq.com
  * @date 2020-3-23 13:33
  *       循环守卫
  */
object Demo02 {
  def main(args: Array[String]): Unit = {

    for (i <- 1 to 3 if i != 2) {
      println("循环守卫i" + i)
    }

    for (j <- 1 to 3) {
      if (j != 2) {
        println("循环守卫j" + j)
      }
    }
  }
}

File: day06/src/com/oxysa/exercise/Demo02.java
package com.oxysa.exercise;

import java.io.BufferedWriter;
import java.io.FileWriter;
import java.util.ArrayList;

/**
 * @author Yummy
 * @mail 873387751@qq.com
 * @date 2020-3-20 16:17
 * 案例: 把ArrayList<String>集合中的内容写入到文件中.
 */
public class Demo02 {
    public static void main(String[] args) throws Exception {
        ArrayList<String> list = new ArrayList<>();
        list.add("小松菜奈");
        list.add("石原里美");
        list.add("桥本环奈");
        list.add("斋藤飞鸟");
        BufferedWriter bw = new BufferedWriter(new FileWriter("./day06/data/4.txt"));
        for (String s : list) {
            bw.write(s);
            bw.newLine();
        }
        System.out.println("写入完成！！！");
        bw.close();
    }
}
